internal/tss/utils/operation_builder_test.go

```diff
@@ -0,0 +1,382 @@
+package utils
+
+import (
+	"encoding/base64"
+	"strings"
+	"testing"
+
+	xdr3 "github.com/stellar/go-xdr/xdr3"
+	"github.com/stellar/go/keypair"
+	"github.com/stellar/go/txnbuild"
+	"github.com/stellar/go/xdr"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBuildOperations(t *testing.T) {
```

Comment:
should there be a case for `t.Run("op_invoke_host_function"...`?

Reply:
+1